python-lib/nlp_utils.py

```diff
@@ -3,16 +3,45 @@
 import unicodedata
 
 
-def get_keyword(text: AnyStr, normalize_case: bool) -> AnyStr:
-    """Return text in its wanted-case form"""
-    return text.lower() if normalize_case else text
+def normalize_case_text(text: AnyStr, lowercase: bool) -> AnyStr:
+    """Return text in its wanted case form"""
+    return text.lower() if lowercase else text
 
 
-def get_sentence(span: Span, normalize_case: bool) -> Union[Span, Doc]:
-    """Return Span object as a Doc if case_insensitive is set to True"""
-    return span if normalize_case else span.as_doc()
+def get_token_attribute(lemmatize: bool) -> AnyStr:
```

Comment:
Nitpick: Would rename to something that is closer to its usage, like `get_phrase_matcher_attr` or simply inline it as it is straighforward, short, and not used so many times

Reply:
Good idea ! Did it in commit 8b8549a